custom_components/plugwise/manifest.json

```
@@ -1,9 +1,9 @@
 {
   "domain": "plugwise",
   "name": "Plugwise Beta",
-  "version": "0.14.6",
+  "version": "0.15.0b1",
```

Current release is already at version 0.15.0.b1, so this PR should bump it at least to a newer version:
```suggestion
  "version": "0.15.0b2",
```

---

I've made a beta release of this b1-code-version so Tom and I (and possible others) could install and test it via HACS.
That's why this PR and the present release have the same version. 
The intention is to merge this PR when the version nr. is 0.15.0, when it is no longer a beta-version.